backend/services/project_service.py

```diff
@@ -108,11 +108,11 @@ def get_contribs_by_day(project_id: int) -> ProjectContribsDTO:
             )
             .group_by("action_text", "day", "task_id")
             .order_by("day")
-        )
+        ).all()
 
         contribs_dto = ProjectContribsDTO()
         # Filter and store unique dates
-        dates = list(set(r[1] for r in stats))
+        dates = [r[1] for r in stats]
```

Comment:
Set collection ensures that there are no duplicates - so in this branch with the current change, there are multiple entries for the same date like here:

```
{
  "stats": [
    {
      "date": "2017-09-20",
      "mapped": 406,
      "validated": 51,
      "cumulative_mapped": 406,
      "cumulative_validated": 51,
      "total_tasks": 1358
    },
    {
      "date": "2017-09-20",
      "mapped": 7,
      "validated": 0,
      "cumulative_mapped": 413,
      "cumulative_validated": 51,
      "total_tasks": 1358
    },
    {
      "date": "2017-09-20",
      "mapped": 0,
      "validated": 0,
      "cumulative_mapped": 413,
      "cumulative_validated": 51,
      "total_tasks": 1358
    },
......
````

Instead it has to be:

```
{
  "stats": [
    {
      "date": "2017-09-20",
      "mapped": 406,
      "validated": 51,
      "cumulative_mapped": 406,
      "cumulative_validated": 51,
      "total_tasks": 1358
    },
    {
      "date": "2017-09-21",
      "mapped": 232,
      "validated": 64,
      "cumulative_mapped": 638,
      "cumulative_validated": 115,
      "total_tasks": 1358
    },
    {
      "date": "2017-09-22",
      "mapped": 85,
      "validated": 3,
      "cumulative_mapped": 723,
      "cumulative_validated": 118,
      "total_tasks": 1358
    },
.....
````